<launch>

<include file="$(find pointgrey_camera_driver)/launch/camera.launch"/>

<!--<include file="$(find usb_cam)/launch/usb_cam-test.launch"/>-->
<!--<include file="$(find ueye_cam)/launch/bayer_rggb_image_proc.launch"/>-->

<node pkg="marker_publisher" type="markerpub" name="markerpub" output="screen" >
      <param name="TheCameraParameters_path" value="$(find marker_publisher)/config/intrinsics.yml" /> <!--camera param-->
      <param name="camera_frame" value="camera_frame" /> <!--camera_frame ref-->
      
      <param name="markerSizeMeters" value="0.117" /> <!--Marker Size in meter-->
      <param name="dict_type" value="TAG36h11" />  
      <!--ARUCO ARUCO_MIP_16h3 ARUCO_MIP_25h7 ARUCO_MIP_36h12 ARTOOLKITPLUS ARTOOLKITPLUSBCH TAG16h5 TAG25h7 TAG25h9 TAG36h11 TAG36h10 CHILITAGS ALL_DICTS-->
      
      <!--<param name="marker_21" value="0.202" /> Marker_id=12 Size meter-->
      <!--<param name="marker_6" value="0.101" /> Marker_id=6 Size meter-->
      <!--<param name="marker_i" value="mi" /> Marker_i Size meter-->
      <!--<param name="marker_j" value="mj" /> Marker_j Size meter-->

      <remap from="/markerpub/image_raw" to="/camera/image_raw" />
</node>

</launch>






